include/oneapi/tbb/concurrent_queue.h

```diff
@@ -26,7 +26,7 @@
 
 namespace tbb {
 namespace detail {
-namespace d1 {
+namespace d2 {
```

Comment:
Not sure I understand why this breaking change is allowed, but others are not. Could someone please explain?

Reply:
Recently, similar changes have been made for concurrent_map and concurrent_set Pull Request #400 
Probably, It's better to schedule a meeting to discuss guidelines for API breaking changes.

Reply:
@aleksei-fedotov as correctness problem fix, we _have_ to do.

Reply:
We are changing the namespace here to prevent additional break of user code if the queue is accessed through the translation unit border. In that case user needs to recompile the both translation units. If the queue is not accessed in different translation units then there is no any breaking change. In any case, the user code is already broken and should be recompiled.
In short, the idea of changed namespace is to prevent additional issues (not break partial recompilation use case but enforce the whole application recompilation).

Reply:
So, the difference is in the correctness problem being fixed. This only fact justifies the recompilation. Okay, I see. Thanks for the explanation!